social/apps/django_app/default/models.py

```diff
@@ -96,6 +96,9 @@ class Association(models.Model, DjangoAssociationMixin):
 
     class Meta:
         db_table = 'social_auth_association'
+        unique_together = (
```

Comment:
Yes, they behave similarly. `unique_together` takes the extra step of enforcing uniqueness in addition to the indexing.